source/aggregation-pipeline-builder.txt

```diff
@@ -207,6 +207,18 @@ your chosen view mode:
             stage card with a form that corresponds with the selected 
             aggregation pipeline stage.
 
+            You can use the following aggregation stages in the Stage Wizard:
```

Comment:
[discussion] To your point, I suspect that adding much more information here would crowd the page. If the info is available in the UI, I think that's fine. If we really need this info in the docs, I'd suggest it might be time to break each of these ways to build an Aggregation Pipeline out into its own page.

Reply:
Noted. We'll see what Julia thinks about this.

Reply:
@mdb-ashley Julia said this looks good as is!